patches/third_party-blink-renderer-bindings-scripts-web_idl-composition_parts.py.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/third_party/blink/renderer/bindings/scripts/web_idl/composition_parts.py b/third_party/blink/renderer/bindings/scripts/web_idl/composition_parts.py
+index 5e8de89409976528f8d97d4962f9f111c6d158a1..d8eca553b5c3aaa2ae3f8644457d20a7c4bb399b 100644
+--- a/third_party/blink/renderer/bindings/scripts/web_idl/composition_parts.py
++++ b/third_party/blink/renderer/bindings/scripts/web_idl/composition_parts.py
+@@ -147,6 +147,9 @@ class Location(object):
+     _blink_path_prefix = posixpath.sep + posixpath.join(
+         'third_party', 'blink', 'renderer', '')
+ 
++    _brave_path_prefix = posixpath.sep + posixpath.join(
```

Comment:
patches should always be on a single line and no blank lines should be added

Reply:
@mkarolin Has fixed this in https://github.com/brave/brave-core/commit/8b6b129d97f7eb820c6fc09418c69625aac6b4be